### PR TITLE
Add Leap 15.3 Maintenance configuration

### PR DESCRIPTION
**Pull request by foursixnine**
Following: https://progress.opensuse.org/issues/72202

For now I don't know how to test it, so directions would be great :)

### CODE REVIEW

**Comment by foursixnine** [comment]
@msmeissn is there something else needed? /cc @okurz

**Comment by msmeissn** [comment]
its not as easy.

you need 2 more repos:

at least:

http://download.opensuse.org/update/leap/15.3/backports/
this is needed definitely.

and also the sle import repos, but I am not sure if these are covered by:

http://download.opensuse.org/update/leap/15.3/sle/

these seem not yet be filled.

**Comment by okurz** [comment]
> its not as easy.
> 
> you need 2 more repos:

Although it would be nice to see if tests would reveal these as missing :)

**Comment by okurz** [comment]
So what I suggest is to have this merged and see what is missing keeping the above note in mind. Note that I don't have permissions to merge, others can

File: data/repos.json [hunk 1]
```diff
@@ -28,5 +28,21 @@
             "http://download.opensuse.org/update/leap/15.2/non-oss/"
          ]
       }
-   }
+   },
+   "openSUSE:Leap:15.3:Update" : {
+         "settings" : {
+            "OS_TEST_ISSUES" : "",
```
Comment (foursixnine) [inline data/repos.json:34]: I wonder if this is actually needed

**Comment by foursixnine** [comment]
@mimi1vx Since Marcus is on Parental leave, do you know if this is all that's needed to enable maintenance repos for 15.3? (if so, can you merge? :P)

**Pull request merged**